tests/unit_test/data_processor/executor_processor_test.py

```
@@ -0,0 +1,401 @@
+import os
+
+import pytest
+from bson import ObjectId
+from mongoengine import connect
+
+from kairon.shared.constants import EventClass
+from kairon.shared.data.constant import EVENT_STATUS, TASK_TYPE
+from kairon.shared.events.processor import ExecutorProcessor
+from kairon.shared.utils import Utility
+
+os.environ["system_file"] = "./tests/testing_data/system.yaml"
```

Ruff

Use capitalized environment variable `SYSTEM_FILE` instead of `system_file`

Replace `system_file` with `SYSTEM_FILE`

(SIM112)